### PR TITLE
Install dssp=3.0 during CI

### PR DESCRIPTION
**Pull request by pckroon**
The github CI runner changed to 22.04 (from 20.04), which has DSSP version 4.0 (https://packages.ubuntu.com/jammy/dssp), which doesn't work with vermouth. So try to specify a different version. If this works we can add a version check to the DSSP processor.

### CODE REVIEW

**Comment by pckroon** [comment]
The only version available to Ubuntu22.04 is 4.0.4-1.

**Comment by pckroon** [comment]
Ok, lgtm. Feel free to approve and merge.

**Comment by pckroon** [comment]
Actually, also set the ubuntu version in the deploy.yaml. Otherwise deployment will still be broken

**Review by pckroon — COMMENTED** [review]
approved. Github thinks this is my PR, so I can't GH approve it.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
           **/pyproject.toml
     - name: Install dependencies part I
       run: |
-        sudo apt-get install dssp
+        sudo apt-get install dssp=3.0.0-3build1
```
Comment (fgrunewald) [inline .github/workflows/deploy.yml:48]: do we want to keep this?

**Review by fgrunewald — APPROVED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (pckroon) [reply .github/workflows/deploy.yml:48]: Meh. Don't particularly care either way, but probably not

**Review by fgrunewald — APPROVED** [review]

**Pull request merged**